### PR TITLE
Feature 68 change active exhibition

### LINKED ISSUE #68: Change active exhibition

### LINKED ISSUE #70: Layout text element alignment

### LINKED ISSUE #71: Bug: Deleting layout component deletes its siblings

### LINKED ISSUE #72: Bug: Clearing layout component proportion results in NaN values

### LINKED ISSUE #73: Bug: Content editor screen lists all devices in an exhibition

### PR DESCRIPTION
**Pull request by villejuutila**
Resolves #68 
Resolves #70 
Resolves #71 
Resolves #72
Resolves #73

### CODE REVIEW

File: src/components/layout/v2/text-component-properties.tsx [hunk 1]
```diff
@@ -186,6 +266,19 @@ const TextComponentProperties = ({
         </Stack>
       </PropertyBox>
       <Divider sx={{ color: "#F5F5F5" }} />
+      <PropertyBox>
+        <PanelSubtitle subtitle="Tekstin tasaus" />
```
Comment (Jdallos) [inline src/components/layout/v2/text-component-properties.tsx:270]: Should the subtitle be localized?

File: src/components/screens/device-models-screen.tsx [hunk 2]
```diff
@@ -154,17 +153,19 @@ export class DeviceModelsScreen extends React.Component<Props, State> {
...
+    const cards = [...deviceModels]
+      .sort((a, b) => a.manufacturer.localeCompare(b.manufacturer))
+      .map((deviceModel) => {
+        return (
+          <CardItem
+            key={deviceModel.id}
+            title={`${deviceModel.manufacturer} ${deviceModel.model}`}
+            onClick={() => this.onCardClick(deviceModel)}
+            menuOptions={this.getCardMenuOptions(deviceModel)}
+            status={""}
```
Comment (Jdallos) [inline src/components/screens/device-models-screen.tsx:165]: Can this prop just be removed, its optional and is typed "small" | "large"

**Review by Jdallos — APPROVED** [review]

File: src/components/layout/v2/text-component-properties.tsx (hunk 1 above)
Comment (villejuutila) [reply src/components/layout/v2/text-component-properties.tsx:270]: Resolved.

File: src/components/screens/device-models-screen.tsx (hunk 2 above)
Comment (villejuutila) [reply src/components/screens/device-models-screen.tsx:165]: Resolved.

**Pull request merged**